convert_payroll_export.py

```diff
@@ -20,11 +20,10 @@ def read_file(file_path: str) -> list:
         return lines
 
 
-def clean_txt (file):
+def clean_file(file):
     # clean txt file or create new file if file not exist
-    file1 = open(file, 'w+')
-    file1.writelines("")
-    file1.close()
+    with open(file, 'w+') as f:
+        f.writelines("")
```

Comment:
If your intention in this case is just remove all the content from the file, you can do it simpler as:
`open("file", "w").close()`